apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -122,29 +122,14 @@ export function InstructorQuestionSettings({
               <form name="edit-question-settings-form" method="POST">
                 <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
                 <input type="hidden" name="orig_hash" value="${origHash}" />
-                <div class="form-group">
-                  <h2 class="h4">General</h2>
-                  <label for="title">Title</label>
-                  <input
-                    type="text"
-                    class="form-control"
-                    id="title"
-                    name="title"
-                    value="${resLocals.question.title}"
-                    ${canEdit ? '' : 'disabled'}
-                  />
-                  <small class="form-text text-muted">
-                    The title of the question (e.g., "Add two numbers").
-                  </small>
-                </div>
                 <div class="form-group">
                   <label for="qid">QID</label>
                   ${questionGHLink
                     ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
                     : ''}
                   <input
                     type="text"
-                    class="form-control"
+                    class="form-control text-monospace"
```

Comment:
Two opportunistic changes here:

- I added `text-monospace` to the QID input
- I reordered the QID input to be first

Both of these changes make this page more consistent with the assessment settings page.